src/Adapter/Zend.php

```diff
@@ -343,6 +343,10 @@ private function _mapData(Entity_Abstract $entity) {
         // entity -> database
         $data = [];
         foreach ($entity->getDirty() as $property => $value) {
+            // If you pass a value for a `on update CURRENT_TIMESTAMP` column, the column won't update. So we're
+            // never letting you accidentally/intentionally send an update to this column.
+            if (in_array($property, [ 'updatedDate', 'updatedAt' ])) continue;
```

Comment:
o the properties in the entities are always camel cased and the corresponding column name in the database table is snake cased